packages/docs/operations-manual/attestation-service.md

```diff
@@ -0,0 +1,311 @@
+```
+
+Lines beginning `#` are treated as comments. In addition, any options specified as environment variables override values provided in this file.
+
+Required options:
+
+| Variable                       |    |
+|--------------------------------|-------------------------------------------------------------------------------------------------|
+| `DATABASE_URL`                   | The URL to access the local database, e.g. `sqlite://db/attestations.db` |
+| `CELO_PROVIDER`                  | The node URL for your local full node at which your attestation signer key is unlocked. e.g. `http://localhost:8545`. Do not expose this port to the public internet! |
+| `CELO_VALIDATOR_ADDRESS`         | Address of the Validator account |
+| `ATTESTATION_SIGNER_ADDRESS`     | Address of the Validator's attestation signer key  |
+| `SMS_PROVIDERS`                  | Comma-separated list of all enabled SMS providers, by order of preference. Can include `twilio`, `nexmo` |
+
+Optional environment variables:
+
+| Variable                       | Explanation    |
+|--------------------------------|-------------------------------------------------------------------------------------------------|
+| `PORT`                           | Port to listen on. Default `3000`. |
+| `SMS_PROVIDERS_<country>`        | Override to set SMS providers and order for a specific country code (e.g `SMS_PROVIDERS_MX=nexmo,twilio`) |
+| `MAX_PROVIDER_RETRIES`           | Number of retries (after first) when sending SMS before considering next provider Default `2`.  |
+| `EXTERNAL_CALLBACK_HOSTPORT`     | Provide the external URL at which providers can attempt callbacks with delivery receipts. If not provided, defaults to the value retrieved from the validator metadata. |
+| `TIMEOUT_CLEANUP_NO_RECEIPT_MIN` | If a delivery report appears to be supported but is not received within this number of minutes, assume delivery success                                               |
+| `VERIFY_CONFIG_ON_STARTUP`       | Refuse to start if signer or metadata is misconfigured. Default `1`. |
+| `LOG_LEVEL`                      | One of `fatal`, `error`, `warn`, `info`, `debug`, `trace` |
+| `LOG_FORMAT`                     | One of `json`, `human`, `stackdriver`  |
+
+Twilio configuration options:
+
+| Variable                       | Explanation                                                     |
+| ------------------------------ | --------------------------------------------------------------- |
+| `TWILIO_ACCOUNT_SID`           | The Twilio account ID                                           |
+| `TWILIO_MESSAGING_SERVICE_SID` | The Twilio Message Service ID. Starts with `MG`                 |
+| `TWILIO_AUTH_TOKEN`            | The API authentication token                                    |
+| `TWILIO_UNSUPPORTED_REGIONS`   | Optional. A comma-separated list of country codes to not serve  |
```

Comment:
example?  is this suppose to be alpha country codes or phone number country codes?
"US" or "+1"